src/Parser/Hunter/BeastMastery/Modules/Spells/AzeriteTraits/HazeOfRage.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import {formatNumber, formatPercentage} from 'common/format';
+import {calculateAzeriteEffects} from 'common/stats';
+import SPELLS from 'common/SPELLS';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'Interface/Others/TraitStatisticBox';
+
+
+const hazeOfRageStats = traits => Object.values(traits).reduce((obj, rank) => {
+  const [agility] = calculateAzeriteEffects(SPELLS.HAZE_OF_RAGE.id, rank);
+  obj.agility += agility;
+  return obj;
+}, {
+  agility: 0,
+});
+
+export const STAT_TRACKER = {
+  agility: combatant => hazeOfRageStats(combatant.traitsBySpellId[SPELLS.HAZE_OF_RAGE.id]).agility,
+};
+
+/**
+ * Bestial Wrath increases your Agility by 376 for 8 sec.
+ *
+ * Example report: https://www.warcraftlogs.com/reports/m9KrNBVCtDALZpzT#boss=-2&difficulty=0&wipes=1&source=5&type=summary
+ */
+class HazeOfRage extends Analyzer{
```

Comment:
A small style note: prefer to put a single space before the opening `{` of a block.